src/index.js

```diff
@@ -1,9 +1,12 @@
 import "./style.css";
-import "./modules/api.js";
-import "./modules/api2.js";
-import { addDummyCards } from "./modules/dummyCards.js";
+import "./modules/apiMovies.js";

```

Comment:
better if you could export the functions and call them in the main rather than excuting them on import for code reusability incase you nede the function twice in your code base